### PR TITLE
Added `ColorPicker` and `ColorSelector` components

### LINKED ISSUE #4002: Separation of UI and Logic in the `ColorPicker` Component
### Description

Separate logic into separate files such as `use-xxx.ts`.
In addition, modify the hooks to function on their own.

### Additional Information

Create a new branch from `v2.0` branch for development.

### LINKED ISSUE #5073: Update documentation for "ColorPicker"
script: update-documentation
page: /docs/components/color-picker

### Path

`/docs/components/color-picker`

### Description

Since the major version is increasing, the documentation will also be updated.

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #5073 
Closes #4002

## Description

Added `ColorPicker` and `ColorSelector` components.

## Is this a breaking change (Yes/No):

No

<h2>Greptile Overview</h2>

Updated On: 2025-10-23 01:35:17 UTC

### Greptile Summary

**This review covers only the changes made since the last review, not the entire PR.** The latest incremental commit addresses all previously flagged test quality and documentation issues across eight files. The changes systematically correct a grammatical typo in placeholder text ("Choice a option/color" → "Choose a option/color") in test files for Autocomplete, ColorPicker, ColorSelector, DatePicker, NativeSelect, and Select components. Additionally, the ColorSelector test file's describe block was corrected from `<Container />` to `<ColorSelector />`, and two documentation files (color-selector.mdx and color-selector.ja.mdx) now correctly import `ColorSelector` instead of the previously incorrect `ColorPicker` references. These are purely non-functional, quality-of-life improvements that enhance test readability and documentation accuracy without affecting runtime behavior, component logic, or test coverage.

<details><summary>Changed Files</summary>

| Filename | Score | Overview |
|----------|-------|----------|
| packages/react/src/components/autocomplete/autocomplete.test.tsx | 5/5 | Fixed placeholder text from "Choice a option" to "Choose a option" in three test cases (lines 7, 27, 57) |
| packages/react/src/components/color-picker/color-picker.test.tsx | 5/5 | Corrected placeholder typo from "Choice a color" to "Choose a color" in all three test cases (lines 6, 17, 32) |
| packages/react/src/components/color-selector/color-selector.test.tsx | 5/5 | Fixed describe block name from `<Container />` to `<ColorSelector />` for accurate test suite identification |
| packages/react/src/components/date-picker/date-picker.test.tsx | 5/5 | Corrected placeholder text from "Choice a option" to "Choose a option" in two test cases (lines 17, 34) |
| packages/react/src/components/native-select/native-select.test.tsx | 5/5 | Fixed placeholder typo from "Choice a option" to "Choose a option" in one test case (line 8) |
| packages/react/src/components/select/select.test.tsx | 5/5 | Corrected placeholder text from "Choice a option" to "Choose a option" in three test cases and corresponding assertions (lines 7, 28, 44, 64,80) |
| www/contents/components/(components)/color-selector.mdx | 5/5 | Fixed import statements to correctly reference `ColorSelector` instead of `ColorPicker` (lines 16-26) |
| www/contents/components/(components)/color-selector.ja.mdx | 5/5 | Corrected import statements from `ColorPicker` to `ColorSelector` (lines 16-26) |

</details>

### Confidence score: 5/5

- This incremental review covers only minor test quality and documentation corrections with zero risk to production behavior or component functionality
- All changes are non-breaking, non-functional improvements that successfully address all previously flagged issues from earlier review iterations
- No files require special attention – all changes are straightforward typo corrections and naming fixes that complete the quality improvement pass across the test suite and documentation

### CODE REVIEW

File: packages/react/src/components/color-picker/color-picker.test.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { a11y, render, screen } from "#test"
+import { ColorPicker } from "."
+
+describe("<ColorPicker />", () => {
+  test("renders component correctly", async () => {
+    await a11y(<ColorPicker placeholder="Choice a color" />)
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/components/color-picker/color-picker.test.tsx]: Fixed.

File: packages/react/src/components/color-picker/color-picker.test.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    expect(screen.getAllByRole("combobox")[0]).toHaveClass(
+      "ui-color-picker__field",
+    )
+  })
+
+  test("renders HTML tag correctly", () => {
+    render(
+      <ColorPicker
+        defaultOpen
+        placeholder="Choice a color"
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/components/color-picker/color-picker.test.tsx]: Fixed.

File: packages/react/src/components/color-selector/color-selector.test.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
+import { render, screen } from "#test"
+import { ColorSelector } from "./"
+
+describe("<Container />", () => {
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/components/color-selector/color-selector.test.tsx]: Fixed.

File: www/contents/components/(components)/color-selector.ja.mdx [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+import { ColorPicker } from "@yamada-ui/react"
+```
+
+```tsx [alias]
+import { ColorPicker } from "@/components/ui"
+```
+
+```tsx [monorepo]
+import { ColorPicker } from "@workspaces/ui"
+```
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply www/contents/components/(components)/color-selector.ja.mdx:26]: Fixed.

File: packages/utils/src/color.test.ts [hunk 5]
```diff
@@ -80,27 +80,27 @@ describe("Color", () => {
   })
 
   test("parseToHsla returns correct fallback color", () => {
-    const [h, s, l, a] = parseToHsla("unknown-color", "#008000") ?? []
+    const [h, s, l, a] = parseToHsla("unknown-color", "#008000")
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/utils/src/color.test.ts:83]: Fixed.

File: packages/react/src/components/color-selector/color-selector.style.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      gap: "1.5",
+      gridTemplateColumns: "repeat({color-swatch-group-columns}, 1fr)",
+    },
+    colorSwatchGroupLabel: {
+      fontSize: "sm",
+      fontWeight: "medium",
+      _disabled: { layerStyle: "disabled" },
+    },
+    colorSwatchItem: { w: "full", _disabled: { layerStyle: "disabled" } },
+    eyeDropper: nativeSelectStyle.base?.icon,
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/components/color-selector/color-selector.style.ts:18]: It is always defined.

File: www/contents/components/(components)/color-selector.mdx [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+import { ColorPicker } from "@yamada-ui/react"
+```
+
+```tsx [alias]
+import { ColorPicker } from "@/components/ui"
+```
+
+```tsx [monorepo]
+import { ColorPicker } from "@workspaces/ui"
+```
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply www/contents/components/(components)/color-selector.mdx:26]: Fixed.

File: www/contents/components/(components)/color-selector.ja.mdx [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+### 読み取り専用にする
+
+```tsx preview
+<ColorSelector.Root maxW="md" readOnly />
+```
+
+### コンポーネントをカスタマイズする
+
+```tsx preview functional client
+const fallbackValue = "#FF0000FF"
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply www/contents/components/(components)/color-selector.ja.mdx]: Fixed.

File: www/contents/components/(components)/color-selector.ja.mdx [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+const format = "hexa"
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply www/contents/components/(components)/color-selector.ja.mdx]: Fixed.

**Review by bmthd — DISMISSED** [review]

File: www/data/props/color-selector.json [hunk 10]
```diff
@@ -0,0 +1,1466 @@
...
+      },
+      "value": {
+        "type": "string",
+        "description": "The value of the color selector.",
+        "required": false
+      },
+      "withAlpha": {
+        "type": "boolean",
+        "defaultValue": "false",
+        "description": "If true, the alpha slider will be displayed.",
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply www/data/props/color-selector.json]: Fixed.

File: www/data/props/color-selector.json [hunk 11]
```diff
@@ -0,0 +1,1466 @@
...
+    },
+    "value": {
+      "type": "string",
+      "description": "The value of the color selector.",
+      "required": false
+    },
+    "withAlpha": {
+      "type": "boolean",
+      "defaultValue": "false",
+      "description": "If true, the alpha slider will be displayed.",
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply www/data/props/color-selector.json]: Fixed.

File: packages/react/src/components/color-picker/color-picker.tsx [hunk 12]
```diff
@@ -0,0 +1,339 @@
...
+const ColorPickerEyeDropper = withContext<"div", ColorPickerEyeDropperProps>(
+  "div",
+  "eyeDropper",
+)(undefined, ({ children, icon, ...rest }) => {
+  const { getEyeDropperProps } = useComponentContext()
+
+  return getEyeDropperProps({
+    children: icon || children || <PipetteIcon />,
+    ...rest,
+  })
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/components/color-picker/color-picker.tsx:331]: No problem.

File: packages/react/src/providers/i18n-provider/intl/en-US.ts [hunk 13]
```diff
@@ -27,6 +27,12 @@ const data = {
   closeButton: {
     Close: "Close",
   },
+  colorPicker: {
+    "Pick a color": "Pick a color",
+  },
+  colorSelector: {
+    "Pick a color": "Pick a color",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/en-US.ts:35]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/tr-TR.ts [hunk 14]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Kapat",
   },
+  colorPicker: {
+    "Pick a color": "Bir renk seçin",
+  },
+  colorSelector: {
+    "Pick a color": "Bir renk seçin",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/tr-TR.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/it-IT.ts [hunk 15]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Chiudi",
   },
+  colorPicker: {
+    "Pick a color": "Scegli un colore",
+  },
+  colorSelector: {
+    "Pick a color": "Scegli un colore",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/it-IT.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/ro-RO.ts [hunk 16]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Închide",
   },
+  colorPicker: {
+    "Pick a color": "Alege o culoare",
+  },
+  colorSelector: {
+    "Pick a color": "Alege o culoare",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/ro-RO.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/fi-FI.ts [hunk 17]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Sulje",
   },
+  colorPicker: {
+    "Pick a color": "Valitse väri",
+  },
+  colorSelector: {
+    "Pick a color": "Valitse väri",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/fi-FI.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/ko-KR.ts [hunk 18]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "닫기",
   },
+  colorPicker: {
+    "Pick a color": "색상 선택",
+  },
+  colorSelector: {
+    "Pick a color": "색상 선택",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/ko-KR.ts:37]: Structural duplication is essential.

File: packages/react/src/components/color-picker/color-picker.style.ts [hunk 19]
```diff
@@ -0,0 +1,36 @@
+import { defineComponentSlotStyle } from "../../core"
+import { datePickerStyle } from "../date-picker"
+import { nativeSelectStyle } from "../native-select"
+
+export const colorPickerStyle = defineComponentSlotStyle({
+  base: {
+    ...nativeSelectStyle.base,
+    colorSwatch: { w: "1.5em" },
+    content: { maxW: "sm", p: "1" },
+    eyeDropper: datePickerStyle.base?.icon,
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/components/color-picker/color-picker.style.ts:10]: It is always defined.

File: packages/react/src/providers/i18n-provider/intl/sv-SE.ts [hunk 20]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Stäng",
   },
+  colorPicker: {
+    "Pick a color": "Välj en färg",
+  },
+  colorSelector: {
+    "Pick a color": "Välj en färg",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/sv-SE.ts:37]: Structural duplication is essential.

File: packages/react/src/components/color-picker/use-color-picker.ts [hunk 21]
```diff
@@ -0,0 +1,391 @@
...
+      onClick: handlerAll(props.onClick, onEyeDropperClick),
+      onKeyDown: handlerAll(props.onKeyDown, (ev) =>
+        runKeyAction(ev, {
+          Enter: onEyeDropperClick,
+          Space: onEyeDropperClick,
+        }),
+      ),
+    }),
+    [dataProps, interactive, onEyeDropperClick, supportedEyeDropper, t],
+  )
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/components/color-picker/use-color-picker.ts:348]: It's okay if it gets overwritten.

File: packages/react/src/providers/i18n-provider/intl/lv-LV.ts [hunk 22]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Aizvērt",
   },
+  colorPicker: {
+    "Pick a color": "Izvēlieties krāsu",
+  },
+  colorSelector: {
+    "Pick a color": "Izvēlieties krāsu",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/lv-LV.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/no-NO.ts [hunk 23]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Lukk",
   },
+  colorPicker: {
+    "Pick a color": "Velg en farge",
+  },
+  colorSelector: {
+    "Pick a color": "Velg en farge",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/no-NO.ts:37]: Structural duplication is essential.

File: packages/react/src/components/color-selector/color-selector.stories.tsx [hunk 24]
```diff
@@ -0,0 +1,140 @@
...
+
+      <Field.Root label="Hex">
+        <InputGroup.Root>
+          <Input
+            value={value.toLocaleUpperCase()}
+            onBlur={(ev) =>
+              setValue(convertColor(ev.target.value, fallbackValue)(format))
+            }
+            onChange={(ev) => setValue(ev.target.value)}
+          />
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/components/color-selector/color-selector.stories.tsx:125]: No problem.

File: packages/react/src/providers/i18n-provider/intl/es-ES.ts [hunk 25]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Cerrar",
   },
+  colorPicker: {
+    "Pick a color": "Elige un color",
+  },
+  colorSelector: {
+    "Pick a color": "Elige un color",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/es-ES.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/ja-JP.ts [hunk 26]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "閉じる",
   },
+  colorPicker: {
+    "Pick a color": "色を選択する",
+  },
+  colorSelector: {
+    "Pick a color": "色を選択する",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/ja-JP.ts:37]: Structural duplication is essential.

**Review by bmthd — APPROVED** [review]

File: packages/react/src/providers/i18n-provider/intl/zh-TW.ts [hunk 27]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "關閉",
   },
+  colorPicker: {
+    "Pick a color": "選擇顏色",
+  },
+  colorSelector: {
+    "Pick a color": "選擇顏色",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/zh-TW.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/nl-NL.ts [hunk 28]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Sluiten",
   },
+  colorPicker: {
+    "Pick a color": "Kies een kleur",
+  },
+  colorSelector: {
+    "Pick a color": "Kies een kleur",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/nl-NL.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/fr-FR.ts [hunk 29]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Fermer",
   },
+  colorPicker: {
+    "Pick a color": "Choisissez une couleur",
+  },
+  colorSelector: {
+    "Pick a color": "Choisissez une couleur",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/fr-FR.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/zh-CN.ts [hunk 30]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "关闭",
   },
+  colorPicker: {
+    "Pick a color": "选择颜色",
+  },
+  colorSelector: {
+    "Pick a color": "选择颜色",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/zh-CN.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/lt-LT.ts [hunk 31]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Uždaryti",
   },
+  colorPicker: {
+    "Pick a color": "Pasirinkite spalvą",
+  },
+  colorSelector: {
+    "Pick a color": "Pasirinkite spalvą",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/lt-LT.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/et-EE.ts [hunk 32]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Sulge",
   },
+  colorPicker: {
+    "Pick a color": "Vali värv",
+  },
+  colorSelector: {
+    "Pick a color": "Vali värv",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/et-EE.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/ru-RU.ts [hunk 33]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Закрыть",
   },
+  colorPicker: {
+    "Pick a color": "Выберите цвет",
+  },
+  colorSelector: {
+    "Pick a color": "Выберите цвет",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/ru-RU.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/he-IL.ts [hunk 34]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "סגור",
   },
+  colorPicker: {
+    "Pick a color": "בחר צבע",
+  },
+  colorSelector: {
+    "Pick a color": "בחר צבע",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/he-IL.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/cs-CZ.ts [hunk 35]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Zavřít",
   },
+  colorPicker: {
+    "Pick a color": "Vyberte barvu",
+  },
+  colorSelector: {
+    "Pick a color": "Vyberte barvu",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/cs-CZ.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/hu-HU.ts [hunk 36]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Bezárás",
   },
+  colorPicker: {
+    "Pick a color": "Válasszon színt",
+  },
+  colorSelector: {
+    "Pick a color": "Válasszon színt",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/hu-HU.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/sr-RS.ts [hunk 37]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Zatvori",
   },
+  colorPicker: {
+    "Pick a color": "Izaberite boju",
+  },
+  colorSelector: {
+    "Pick a color": "Izaberite boju",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/sr-RS.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/sk-SK.ts [hunk 38]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Zatvoriť",
   },
+  colorPicker: {
+    "Pick a color": "Vyberte farbu",
+  },
+  colorSelector: {
+    "Pick a color": "Vyberte farbu",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/sk-SK.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/pt-BR.ts [hunk 39]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Fechar",
   },
+  colorPicker: {
+    "Pick a color": "Escolha uma cor",
+  },
+  colorSelector: {
+    "Pick a color": "Escolha uma cor",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/pt-BR.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/da-DK.ts [hunk 40]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Luk",
   },
+  colorPicker: {
+    "Pick a color": "Vælg en farve",
+  },
+  colorSelector: {
+    "Pick a color": "Vælg en farve",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/da-DK.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/sl-SI.ts [hunk 41]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Zapri",
   },
+  colorPicker: {
+    "Pick a color": "Izberite barvo",
+  },
+  colorSelector: {
+    "Pick a color": "Izberite barvo",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/sl-SI.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/ar-AE.ts [hunk 42]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "إغلاق",
   },
+  colorPicker: {
+    "Pick a color": "اختر لونًا",
+  },
+  colorSelector: {
+    "Pick a color": "اختر لونًا",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/ar-AE.ts:37]: Structural duplication is essential.

File: packages/react/src/providers/i18n-provider/intl/uk-UA.ts [hunk 43]
```diff
@@ -29,6 +29,12 @@ const data: IntlData = {
   closeButton: {
     Close: "Закрити",
   },
+  colorPicker: {
+    "Pick a color": "Виберіть колір",
+  },
+  colorSelector: {
+    "Pick a color": "Виберіть колір",
+  },
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply packages/react/src/providers/i18n-provider/intl/uk-UA.ts:37]: Structural duplication is essential.
